src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -435,13 +425,13 @@
 
         "toolFingerprint": 
         {
-          "description": "A string that baselining mechanisms can merge with other data to help uniquely identify this issue, run-over-run.",
+          "description": "A string that contributes to the process of uniquely identifying the result.",
           "type": "string"
         },
 
         "stacks": 
         {
-          "description": "A grouped set of locations, if available, that represent stacks associated with this result.",
+          "description": "An array of arrays of 'annotatedCodeLocation' objects, each inner array of which describes a single call stack.",
```

Comment:
"comprises a call stack", I'd prefer, but yes. Fixed here and for code flows. No corresponding language in the spec.